Wikipedia/Code/Article as a Living Document/ArticleAsLivingDocSnippetCollectionViewCell.swift

```diff
@@ -0,0 +1,26 @@
+
+import UIKit
+
+class ArticleAsLivingDocSnippetCollectionViewCell: ArticleAsLivingDocHorizontallyScrollingCell {
+    
+    override func sizeThatFits(_ size: CGSize, apply: Bool) -> CGSize {
+        
+        let adjustedMargins = ArticleAsLivingDocViewModel.Event.Large.padding
+
+        let descriptionX = adjustedMargins.left
+        
+        let descriptionOrigin = CGPoint(x: descriptionX, y: adjustedMargins.top)
```

Comment:
Super, super minor, feel free to ignore: I'm not sure that `descriptionX` clarifies much here. (Maybe it was an artifact from an earlier draft?)

Reply:
Agreed, I'll fix.